frontend/src/pages/AccountSend.vue

```diff
@@ -292,21 +293,60 @@ function useSendForm() {
   }
 
   const isNativeToken = (address: string) => getAddress(address) === NATIVE_TOKEN.value.address;
-  const getMinSendAmount = (tokenAddress: string) => {
+
+  const getNativeTokenMinSendAmountFallback = (chainId: number): number => {
+    switch (chainId) {
+      case 137: return 1.0; // Polygon
+      default: return 0.01; // everything else
+    }
+  }
+
+  const getTokenMinSendAmountFallback = (
+    tokenAddress: string,
+    chainId: number
+  ): number => {
+    switch (chainId) {
+      case 137: // Polygon
+        if (tokenAddress === '0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619') {
+          return 0.001; // weth token minimum
+        } else {
+          return 3; // stablecoin token minimum
+        }
+      default:
+        // Mainnet, Rinkeby, and other networks have higher ETH and stablecoin minimums
+        // due to higher fees
+        return 100
+    }
+  }
+
+  const getMinSendAmount = (tokenAddress: string): number => {
     const chainId = BigNumber.from(currentChain.value?.chainId).toNumber();
```

Comment:
We use `chainId` enough that we should probably just add a `chainId` export from `wallet.ts` and start using that. We can do it as a computed property from `network.value.chainId`. I know this line was unmodified, so we can pull this out to a separate issue if you want